src/rust/cryptography-x509-validation/Cargo.toml

```diff
@@ -11,3 +11,6 @@ rust-version = "1.63.0"
 asn1 = { version = "0.15.5", default-features = false }
 cryptography-x509 = { path = "../cryptography-x509" }
 once_cell = "1"
+
+[dev-dependencies]
+pem = "1.1"
```

Comment:
Old version

Reply:
Actually unused?

Reply:
Yeah, unused now that `NullOps` is removed. I'll remove entirely.

Reply:
[a5a88e6](https://github.com/pyca/cryptography/pull/9405/commits/a5a88e6291c7f85e8a1915d70c618ba407a5a56a)